icu4c/source/test/cintltst/udatatst.c

```diff
@@ -1626,7 +1626,7 @@ TestSwapCase(UDataMemory *pData, const char *name,
 
 static void U_CALLCONV
 printErrorToString(void *context, const char *fmt, va_list args) {
-    vsprintf((char *)context, fmt, args);
+    vsnprintf((char *)context, 100, fmt, args);
```

Comment:
Is there a reason you didn't use `sizeof` here?

Reply:
this is much hader to get the size and I cannot simply use sizeof here

Reply:
Well, it's unit-test code, so I'm not terribly concerned.